### DISCUSSION TITLE
Update branch button available when it can't be used

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#139954] [Mobile] [answered]
GitHub Mobile beta for Android 

1. Open https://github.com/traefik/traefik/pull/11053
2. Press update branch button

# Actually results

![Screenshot_20240927-092554.png](https://github.com/user-attachments/assets/ecea1026-7a7e-411f-bee6-cd8db69b5017)

> jsoref does not have the correct permissions to execute'UpdatePull RequestBranch

# Expected results after step 1

![Screenshot_20240927-092617.png](https://github.com/user-attachments/assets/e6053df4-ea1f-4f94-b7cd-59cf2c3eace0)

Note the lack of a button -- because I don't have write permission to the base or head repositories.

**Comment by HenSquared** [comment, marked as answer]
@jsoref , thanks for the feedback. I opened an internal issue to track this.

**Reply by Clydonight** [reply]
Thanks alot ill go try it...
